qadence/operations.py

```diff
@@ -1124,6 +1112,30 @@ def wait(
     return WaitBlock(parameters=ps, qubit_support=q, add_pattern=add_pattern)
```

Comment:
It's strange to me that we have `WaitBlock` here, but otherwise renamed it

Reply:
Yes there are quite a few things still to be worked on in this analog part of the block system imo, here I just wanted to deprecate the usage of `wait` in favor of something more explicit and deal with the rest later.

Reply:
But could we not at least also rename that object?
With this change we'll have
```
>>> print(AnalogInteraction(10))
WaitBlock(t=10.0, support=(<QubitSupportType.GLOBAL: 'global'>,))
```
Moreover, if you print the diagram of the circutit it will also be named "Wait"

Reply:
If theres a decent reason not to, feel free to merge, but let's not postpone much finalizing it?

Reply:
Yes indeed I think it can be renamed